tests.html

```diff
@@ -245,7 +250,7 @@ <h2 id="g2-title"></h2>
                         ORDER BY run DESC
                         LIMIT 2`);
                     const links = urls[0].values.map(u => u[0]);
-                    top10.push([{name: row[0], key: "test"}, row[1].toFixed(2), links]);
+                    top10.push([{name: normalize(row[0]), key_name: row[0], key: "test"}, row[1].toFixed(2), links]);
```

Comment:
Any chance we can add the project here? check-application is rather vague, our current top failures are from cockpit-certificates. The new selector surely helps with that, though!

Reply:
Good idea! Pushed additional commit. (And deployed it on my page as well for reviewing).